File: system-test/dotnet/SystemTests/Legacy/Mod04/SmokeTests/External/TaxSmokeTest.cs
using Common;
using SystemTests.Legacy.Mod04.Base;
using Xunit;

namespace SystemTests.Legacy.Mod04.SmokeTests.External;

public class TaxSmokeTest : BaseClientTest
{
    public override Task InitializeAsync()
    {
        SetUpExternalClients();
        return Task.CompletedTask;
    }

    [Fact]
    public async Task ShouldBeAbleToGoToTax()
    {
        var result = await _taxClient!.CheckHealthAsync();
        result.ShouldBeSuccess();
    }
}

File: system-test/java/src/test/java/com/optivem/shop/systemtest/legacy/mod04/smoke/external/TaxSmokeTest.java
package com.optivem.shop.systemtest.legacy.mod04.smoke.external;

import com.optivem.shop.systemtest.legacy.mod04.base.BaseClientTest;
import org.junit.jupiter.api.BeforeEach;
import org.junit.jupiter.api.Test;

import static com.optivem.shop.testkit.common.ResultAssert.assertThatResult;

class TaxSmokeTest extends BaseClientTest {
    @BeforeEach
    void setUp() {
        setUpExternalClients();
    }

    @Test
    void shouldBeAbleToGoToTax() {
        var result = taxClient.checkHealth();
        assertThatResult(result).isSuccess();
    }
}
